contracts/cloud.yml

```diff
@@ -38,8 +37,10 @@ tags:
 
       #### User sessions with authorizations
 
-      If a user signs in with username and password, creating a _user session_,
+      Optionally, when creating an authorization, you can scope it to a specific user.
+      If the user signs in with username and password, creating a _user session_,
       the session carries the permissions granted by all the user's authorizations.
+      For more information, see [how to assign a token to a specific user](https://docs.influxdata.com/influxdb/cloud/security/tokens/create-token/).
```

Comment:
```suggestion
      For more information, see [how to assign a token to a specific user](/influxdb/cloud/security/tokens/create-token/).
```

Reply:
> 
You only need to review the `src/` files, not the `contracts` for openapi.

This is in the generated contract (after `{{% INFLUXDB_DOCS_URL %}}` is subbed). On the docs side, `api/docs/getswagger.sh` makes your change.